.github/workflows/test.yml

```diff
@@ -0,0 +1,106 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+      - mv3-chrome
+      - travis-github-action
+      - travis-github-action-mv3
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    continue-on-error: ${{ matrix.job != 'lint' }}
+    strategy:
+      matrix:
+        job: [lint, firefox, firefox-beta, firefox-nightly, firefox-esr, edge-beta, chrome, chrome-beta]
```

Comment:
How does chrome/chrome-beta work with MV2 code? Shouldn't these be absent from the job matrix in MV2 branches?